public/css/utils.css

```diff
@@ -0,0 +1,134 @@
+/* stylelint-disable media-feature-range-notation */
+
+.btn {
+  display: inline-block;
+  padding: 0.5rem 1rem;
+  font-weight: 600;
+  color: var(--clr-slate50);
+  letter-spacing: 0.5rem;
+  text-decoration: none;
+  border-radius: 6px;
+  background-color: var(--clr-button);
+}
+
+.btn:hover {
+  background-color: var(--clr-button-hover);
+}
+
+/* xs */
+@media screen and (max-width: 400px) {
+  .container-header {
+    max-width: 93%;
```

Comment:
[Q]
Where is this number coming from? 👀

Reply:
If you are reffing to ` max-width: 93%;` it was what I was forced to due to my limited experience.
I could not get it right any other way! The header was the one element that took the longest trial and error
from mobile to desktop.
or if you are referring to `max-width: 400px` this is the screen breakpoint where the changes start taking effect.